src/components/KantoSVG.html

```diff
@@ -392,14 +392,14 @@
 
     <!--Celadon City-->
     <%- include('templates/mapTown',
-        { name: "Celadon City"
+        { name: "Saffron City"
         , x: 59, y: 21
         })
      %>
 
     <!--Saffron City-->
     <%- include('templates/mapTown',
-        { name: "Saffron City"
+        { name: "Celadon City"
```

Comment:
Probably better to swap the co-ords rather than the names, as the comments also need to be changed otherwise, and then it stays in the same order.

Reply:
Done